### ISSUE TITLE
problems with pretrained model on ITS dataset

### ISSUE THREAD

**Issue by cxl-ustb** [AmeryXiong/MixDehazeNet#6]
Hello,thank you for your good work.I have a problem,when I test the Indoor pretrain model on ITS train set,its psnr is around 22.But when I test the Indoor pretrain model on SOTS test set,its psnr is around 43.Why?

**Comment by AmeryXiong** [comment]
Because of different test datasets, the test results are different.

**Comment by cxl-ustb** [comment]
> Because of different test datasets, the te
Thank you.which dataset you used when you train the pretrain model MixDehazeNet-l?not RESIDE ITS?

**Comment by AmeryXiong** [comment]
I train the model in RESIDE-ITS dataset and test it in SOTS-Indoor test dataset. You can see the details in my paper.

**Comment by cxl-ustb** [comment]
> I train the model in RESIDE-ITS dataset and test it in SOTS-Indoor test dataset. You can see the details in my paper.

Thank you very much for your reply.I have seen this before. I would like to borrow your training results, which are the results of the model on the training set. But I used your pre trained model to infer the results on the entire training set, and the psnr is only a little over 20. Should the PSNR of your training on the training set not be greater than the results on the test set?

**Comment by AmeryXiong** [comment]
我没有关注过训练集中推理的结果，我认为可能是因为训练集中图片数量过大，导致训练集推理结果不太好。 或许其他模型在训练集中的推理结果也不太好。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
